concordium-contracts-common/src/traits.rs

```diff
@@ -196,7 +196,7 @@ impl Write for &mut [u8] {
     #[inline]
     fn write(&mut self, buf: &[u8]) -> Result<usize, Self::Err> {
         let to_write = core::cmp::min(buf.len(), self.len());
-        let (overwrite, rest) = core::mem::replace(self, &mut []).split_at_mut(to_write);
```

Comment:
Well the thing is that I prefer using replace since it is more explicit. So why was this, unrelated, thing changed?

Reply:
Clippy was complaining.